torchao/quantization/quant_api.py

```diff
@@ -82,6 +82,7 @@
 from .utils import _get_per_token_block_size
 
 logger = logging.getLogger(__name__)
+is_cuda_8_9 = torch.cuda.is_available() and torch.cuda.get_device_capability() >= (8, 9)
```

Comment:
what about weight only quant, does it need CUDA 8.9 as well

Reply:
For weight only, currently we just dequantize and perform the operation. The reason other techniques are cuda 8.9 dependent, is because scaled_mm is only supported on cuda >= 8.9